File: packages/gbnf-experiment/data/20260909T151143Z_170b737d/ported_implementation/src/grammar_graph/colorize.ts
export const Color = {
  BLUE: "\x1b[34m",
  CYAN: "\x1b[36m",
  GREEN: "\x1b[32m",
  RED: "\x1b[31m",
  GRAY: "\x1b[90m",
  YELLOW: "\x1b[33m",
} as const;

export const colorize = (text: string | number, color: string): string => `${color}${text}`;

File: proposed-projects/round-trip-experiment/reverse/20260909T023701Z_e94dca40/20260910T060603Z_391335fe/ported_implementation/gbnf/grammar_graph/colorize.py
from __future__ import annotations


class Color:
    BLUE = "\x1b[34m"
    CYAN = "\x1b[36m"
    GREEN = "\x1b[32m"
    RED = "\x1b[31m"
    GRAY = "\x1b[90m"
    YELLOW = "\x1b[33m"


def colorize(text: str | int, color: str) -> str:
    return f"{color}{text}"
